docs/tutorials/NEB/neb_tutorial.md

```diff
@@ -273,11 +272,14 @@ The ASE-generated plot looks like this:
 
 ![NEB Plot](../../img/neb.png)
 
+*[Do the green lines have any significance?]*
```

Comment:
ASE NEB plots show these green tangent lines -- they can be useful to see if something is going weird between two images. For this simple path they don't really add much, but it's more work to get rid of them than to keep them.

Reply:
Again, I would just give the 1st sentence.  You just don't want "unexplained stuff" in a tutorial.

Reply:
```suggestion
The green lines show the tangents at those points; these can be useful indicators of convergence. 
```